File: shuffle.js
const { readdirSync, rename } = require('fs');
const { resolve } = require('path');



const imageDirPath = resolve(__dirname, './images/');

console.log(imageDirPath);
const files = readdirSync(imageDirPath);

console.log(files.length)


//files.forEach(seeFiles);

//function seeFiles(item) {
//    console.log(item);
//}
const old = "./images/";





let counter = 1;

while (files.length != 0) {
    let y = Math.floor(Math.random() * (files.length - 1));
    console.log(y);


    rename(old + files[y], old + "" + String(counter) + ".jpg", (err) => {
        if (err) console.log(err)
    })

    counter = counter + 1
    files.splice(y, 1);




}

File: shuffle.py
import random
import os

# names = ["Ashley.jpg", "Kyle.jpg", "Lauren.jpg", "Thomas.jpg", "Michael.jpg", "Dominique.jpg"]
newNames = []

files = os.listdir("./images")
# discard = files.pop(0)
print(files)

old = "./images/"


# for z in files:
#    print(old + z)


counter = 0

"""while len(names) != 0:
    y = random.randint(0,len(names)-1) 
    print(names[y]+', '+str(counter))
    counter = counter+1
    names.pop(y)"""

# next rename each file using the counter

while len(files) != 0:
    y = random.randint(0, len(files) - 1)

    # print(names[y])

    os.rename(old + files[y], old + "00000" + str(counter) + ".jpg")

    # os.rename(old + files[y], new + str(counter) + ".jpg")
    # print(new + str(counter) + ".jpg")
    # os.rename( + files[y], old + str(counter) + ".jpg")
    # files[y] = str(counter)+'.jpg'
    newNames.append(files[y])
    counter = counter + 1
    files.pop(y)

print(newNames)
